tests/test_run_worker.py

```diff
@@ -0,0 +1,94 @@
+# @@@SNIPSTART hello-world-project-template-python-tests
+import uuid
+
+import pytest
+from temporalio.client import WorkflowFailureError
+from temporalio.testing import WorkflowEnvironment
+from temporalio.worker import Worker
+
+from activities import deposit, refund, withdraw
+from shared import PaymentDetails
+from workflows import MoneyTransfer
+
+
+@pytest.mark.asyncio
+async def test_money_transfer() -> None:
+    task_queue_name: str = str(uuid.uuid4())
+    async with await WorkflowEnvironment.start_time_skipping() as env:
+        data: PaymentDetails = PaymentDetails(
+            source_account="85-150",
+            target_account="43-812",
+            amount=250,
+            reference_id="12345",
+        )
+
+        async with Worker(
+            env.client,
+            task_queue=task_queue_name,
+            workflows=[MoneyTransfer],
+            activities=[withdraw, deposit, refund],
```

Comment:
Often a good test would not run real activities against the same bank service they would in non-test (even though technically your bank service is a fake even in non-tests)